packages/components/src/popover/index.js

```diff
@@ -231,11 +231,11 @@ const Popover = (
 			? undefined
 			: size( {
 					apply( sizeProps ) {
-						const { height } = sizeProps;
```

Comment:
Well spotted!

For reference, this breaking change was [introduced in the `0.7.0` version](https://github.com/floating-ui/floating-ui/releases/tag/%40floating-ui%2Fcore%400.7.0)